### PR TITLE
doc: Add a short "about" to front page.

### PR DESCRIPTION
**Pull request by htgoebel**

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -6,6 +6,12 @@
 Welcome to Pytition's documentation!
 ====================================
 
+`Pytition` is a application for privacy-friendly online petitions
+wou can host on your own server.
```
Comment (fallen) [inline doc/index.rst:10]: wou -> you

File: doc/index.rst [hunk 2]
```diff
@@ -6,6 +6,12 @@
 Welcome to Pytition's documentation!
 ====================================
 
+`Pytition` is a application for privacy-friendly online petitions
```
Comment (fallen) [inline doc/index.rst:9]: a -> an

File: doc/index.rst [hunk 3]
```diff
@@ -6,6 +6,12 @@
...
+wou can host on your own server.
+`Pytition` uses the renown `Django` framework and is easy to install.
+
+Homepage and Demo: https://pytitiondemo.sionneau.net/
```
Comment (fallen) [inline doc/index.rst:13]: That's not the homepage but only the Demo website.

**Comment by htgoebel** [comment]
Thanks for the quick review. Looks like I'v been to hasty :-) Should be fixed now. I suggest using a squash merge.

**Pull request merged**